tests/components/unifiprotect/conftest.py

```diff
@@ -202,6 +190,17 @@ def mock_viewer():
     yield Viewer.from_unifi_dict(**data)
 
 
+@pytest.fixture
+def mock_sensor():
+    """Mock UniFi Protect Sensor device."""
+
+    path = Path(__file__).parent / "sample_data" / "sample_sensor.json"
```

Comment:
We have this helper:
https://github.com/home-assistant/core/blob/ae6d9c229e910b2802711c4d7575c076649d08d7/tests/common.py#L423